src/app/(app)/(home)/my-stage-bookings/[id]/page.tsx

```
@@ -0,0 +1,493 @@
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (status === "unauthenticated") {
+      router.push("/sign-in");
+      return;
+    }
+
+    if (status === "authenticated" && params.id) {
+      fetchBooking();
+    }
+  }, [status, router, params.id, fetchBooking]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)